configuration/pih/htmlforms/hiv/section-hiv-plan.xml

```diff
@@ -498,161 +502,200 @@
                             </td>
-                            <td class="other-pro-treat">
-                                <obs conceptId="CIEL:164384" allowFutureDates="true" />
+                            <td>
+                                <label>
+                                    <uimessage code="pihcore.hiv.statusAndOrder" />
+                                </label>
                             </td>
-                            <td class="other-pro-treat">
-                                <obs conceptId="CIEL:159367" answerConceptId="CIEL:1065"
-                                     answerLabel=""/>
+                            <td>
+                                <label>
+                                    <uimessage code="pihcore.hiv.dosage" />
+                                </label>
                             </td>
-                            <td class="other-pro-treat">
-                                <small>
-                                    <obs conceptId="CIEL:6098"
-                                         answerConceptIds="CIEL:843,CIEL:1754,CIEL:102,CIEL:987,CIEL:1434,CIEL:5622" />
-                                </small>
+                            <td>
                             </td>
                         </tr>
-                    </obsgroup>
-                </table>
+
+                        <repeat>
+                            <template>
+                                <obsgroup groupingConceptId="PIH:Prescription construct">
+                                    <tr>
+                                        <td>
+                                            <!-- Medication orders -->
+                                            <obs conceptId="CIEL:1282" answerConceptId="{conceptMap}"
+                                                answerLabel="{drugname}"
+                                                toggle="{id: '{drugname}-treat', style: 'dim'}"/>
+                                        </td>
+                                        <td class="{drugname}-treat">
+                                            <obs conceptId="PIH:PROPHYLAXIS STARTED"
+                                                 answerConceptId="CIEL:1065" style="checkbox"
+                                                 labelCode="pihcore.hiv.started" answerLabel=""
+                                                 toggle="{drugname}-start-date"/>
+                                            <obs conceptId="PIH:Prophylaxis should continue"
+                                                 answerConceptId="CIEL:1065" style="checkbox"
+                                                 labelCode="pihcore.hiv.continue" answerLabel="" />
+                                            <obs conceptId="PIH:Stop Rx" style="checkbox"
+                                                 answerConceptId="CIEL:1065"
+                                                 labelCode="pihcore.hiv.stop" answerLabel=""
```

Comment:
Cannot find a handy translation.  Change labelCode="pihcore.stop".  There's nothing about this specific to HIV.